tests/providers/aws/services/rds/rds_service_test.py

```diff
@@ -249,3 +249,37 @@ def test__describe_db_cluster_snapshots__(self):
         assert rds.db_cluster_snapshots[0].cluster_id == "db-primary-1"
         assert rds.db_cluster_snapshots[0].region == AWS_REGION
         assert not rds.db_cluster_snapshots[0].public
+
+    # Test RDS describe db engine versions
+    @mock_rds
+    def test__describe_db_engine_versions__(self):
+        conn = client("rds", region_name=AWS_REGION)
+        conn.create_db_parameter_group(
+            DBParameterGroupName="test",
+            DBParameterGroupFamily="default.mysql8.0.23",
+            Description="test parameter group",
+        )
+        conn.create_db_instance(
+            DBInstanceIdentifier="db-master-1",
+            AllocatedStorage=10,
+            Engine="mysql",
+            EngineVersion="8.0.23",
+            DBName="staging-mysql",
+            DBInstanceClass="db.m1.small",
+            StorageEncrypted=True,
+            DeletionProtection=True,
+            PubliclyAccessible=True,
+            AutoMinorVersionUpgrade=True,
+            BackupRetentionPeriod=10,
+            EnableCloudwatchLogsExports=["audit", "error"],
+            MultiAZ=True,
+            DBParameterGroupName="test",
+        )
+        # RDS client for this test class
+        audit_info = self.set_mocked_audit_info()
+        rds = RDS(audit_info)
+        assert len(rds.db_instances) == 1
+        # assert rds.db_engines[0].id == "db-master-1"
+        # assert rds.db_engines[0].region == AWS_REGION
+        # assert rds.db_engines[0].engine == "mysql"
+        assert rds.db_instances[0].engine_version == "8.0.23"
```

Comment:
Here you must also check the new `DBEngine` class created and the `self.db_engines` list.